Executable/AudioFixer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using AssetsTools.NET;
+using AssetsTools.NET.Extra;
+using System.Collections.Generic;
+using System.IO;
+
+namespace QModManager
+{
+    internal static class AudioFixer
+    {
+        internal static void ChangeDisableUnityAudio(string path, bool newValue)
```

Comment:
Someone is going to forget what `newValue` is supposed to mean.  
Give this parameter a more meaningful name.

Reply:
The parameter literally means the new value of the `Disable Unity Audio` setting